medusa/search/backlog.py

```diff
@@ -49,6 +49,7 @@ def __init__(self):
         self.amActive = False
         self.amPaused = False
         self.amWaiting = False
+        self.force = False
```

Comment:
If you mean `force` as an attribute (meaning this backlog was triggered as the result of a "forced backlog" action) then the attribute name should be called `forced`.  But as @medariox asked: What do you mean?